### PR TITLE
docs: Remove why page and redirect to OPF

### LINKED ISSUE #1038: docs: Remove why oasis page
Matej confirmed we should delete https://docs.oasis.io/general/oasis-network/why-oasis and redirecting people to oasisprotocol.org.

### PR DESCRIPTION
**Pull request by aefhm**
## Description

Close #1038

## TODO

- [x] Check URL redirect

### CODE REVIEW

File: redirects.ts [hunk 1]
```diff
@@ -183,13 +183,17 @@ export const redirectsOptions: Options = {
                 '/oasis-core-ledger/release-process', // #626 Deprecate oasis-core-ledger
             ]
         },
+        {
+            to: 'https://oasisprotocol.org/',
+            from: '/general/oasis-network/why-oasis', // #1052 Redirect why page to OPF
```
Comment (matevz) [inline redirects.ts]:
```suggestion
            from: [
                '/general/oasis-network/why-oasis', // #1052 Redirect why page to OPF
                '/oasis-network-primer/', // #200 Restructure docs
            ]
```

File: docs/general/oasis-network/README.mdx [hunk 2]
```diff
@@ -49,5 +49,5 @@ The versatility of the ParaTime layer allows the Oasis Network to expand and gro
 ### Learn more
 
 <DocCard item={
-    findSidebarItem('/general/oasis-network/why-oasis')
+    findSidebarItem('/general/oasis-network/token-metrics-and-distribution')
```
Comment (matevz) [inline docs/general/oasis-network/README.mdx:50]: I think we can safely remove the "Learn more" section here.

**Review by matevz — APPROVED** [review]
Some more suggestions. Otherwise looks good.

File: docs/general/oasis-network/README.mdx (hunk 2 above)
Comment (aefhm) [reply docs/general/oasis-network/README.mdx:50]: Ah, you are right.

**Pull request merged**